plugins/woocommerce/client/legacy/js/frontend/add-to-cart-variation.js

```diff
@@ -603,6 +606,16 @@
 			.slideDown( 200 );
 	};
 
+	/**
+	 * Handle reset key down event for accessibility.
+	 */
+	VariationForm.prototype.onResetKeyDown = function ( event ) {
+		if ( event.keyCode === 32 || event.keyCode === 13 ) {
```

Comment:
I would change this to using the `code` property instead `if ( event.code === 'Enter' || event.code === 'Space' ) {`